Price Getter to convert from integer to decimal

cl5rk opened the pull request.

Commit:
price getter working

angelato13 reviewed (commented):
Suggest also adding a setter so it is consistent with getter

Suggested code to add to server/db/models/product.js (price defn):
set(price) {
this.setDataValue('price', price * 100)
}

angelato13 committed:
Update product.js to include price setter

Added working code for setter for price. (Tested locally.) If admin tries to update price to "5", it will push "500" into database.
